src/searchbar/SearchBar-default.js

```diff
@@ -50,7 +50,9 @@ class Search extends Component {
 
```

Comment:
The default SearchBar in `SearchBar-default.js` is already fully customizable. We just need to make the API more uniform between iOS/Android/Default searchbar, but I think you can let this file as it was. You are already able to change the background color.